src/main/webapp/app/overview/exercise-details/exercise-details-student-actions.component.ts

```diff
@@ -72,6 +87,9 @@ export class ExerciseDetailsStudentActionsComponent implements OnInit {
      */
     repositoryUrl(participation: Participation) {
         const programmingParticipation = participation as ProgrammingExerciseStudentParticipation;
+        if (this.useSsh) {
```

Comment:
I think there is a small mistake here. Using ssh should be independent of the mode (individual vs. team).
Otherwise a student might have a problem in a team programming exercise

Reply:
the ssh url is the same for individual and team exercises

Reply:
ok, then we should add this as a comment here, otherwise this might be confusing

Reply:
ok